integration-cli/docker_cli_daemon_test.go

```diff
@@ -1723,3 +1723,30 @@ func (s *DockerDaemonSuite) TestDaemonStartWithoutHost(c *check.C) {
 	}()
 	c.Assert(s.d.Start(), check.IsNil)
 }
+
+func (s *DockerDaemonSuite) TestBridgeIPIsExcludedFromAllocatorPool(c *check.C) {
+	defaultNetworkBridge := "docker0"
+	deleteInterface(c, defaultNetworkBridge)
+
+	bridgeIP := "192.169.1.1"
+	bridgeRange := bridgeIP + "/30"
+
+	err := s.d.StartWithBusybox("--bip", bridgeRange)
+	c.Assert(err, check.IsNil)
+	defer s.d.Restart()
+
+	var cont int
+	for {
```

Comment:
@mavenugo thx for your comments :) the test doesn't know beforehand if the bip is not included in the pool, it tries to verify that. Regarding the subnet and broadcast addresses IMO the objective of the test is clearer if it doesn't make assumptions about the ips used in the range, what do you think?